server/config_unix.go

```diff
@@ -7,3 +7,6 @@ const CrioConfigPath = "/etc/crio/crio.conf"
 
 // CrioSocketPath is where the unix socket is located
 const CrioSocketPath = "/var/run/crio/crio.sock"
+
+// CrioVersionPath is where the CRI-O version file is located
+const CrioVersionPath = "/etc/crio/version"
```

Comment:
I would make this configurable (also in relation to testing), but this definitely only a _nice to have_ in my POV.

Reply:
does this information need to be on persistent storage?  If so we should better use `/var/lib`.  If it is not supposed to be persistent then we should use `/run`.